src/coreclr/src/pal/inc/pal.h

```diff
@@ -1107,13 +1098,7 @@ OpenSemaphoreW(
     IN BOOL bInheritHandle,
     IN LPCWSTR lpName);
 
-#ifdef UNICODE
-#define CreateSemaphore CreateSemaphoreW
-#define CreateSemaphoreEx CreateSemaphoreExW
-#else
-#define CreateSemaphore CreateSemaphoreA
-#define CreateSemaphoreEx CreateSemaphoreExA
-#endif
+#define CreateSemaphore CreateSemaphoreExW
```

Comment:
```suggestion
#if UNICODE
#define CreateSemaphoreEx CreateSemaphoreExW
#endif
```

I think it looks better to preserve the style and keep `#if UNICODE` around these #defines.

Reply:
(same in few other places)

Reply:
This will not work because we are unconditionally using CreateSemaphoreExW